src/index.ts

```diff
@@ -58,8 +58,13 @@ export default class MulterGoogleCloudStorage implements multer.StorageEngine {
 				if (err) {
 					return cb(err);
 				}
-				var gcFile = this.gcsBucket.file(filename);
-				file.stream.pipe(gcFile.createWriteStream({ predefinedAcl: this.options.acl || 'private' }))
+        var gcFile = this.gcsBucket.file(filename);
+				file.stream.pipe(gcFile.createWriteStream({
+          predefinedAcl: this.options.acl || 'private',
+          metadata: {
```

Comment:
Rather than making the metadata a hardcoded value, it should be passed in optionally with `opts`. This way access-control headers and cache-control headers can also be set on the objects being streamed to GCS.

Reply:
good, I agree with that. I will fix it